### PR TITLE
(Reddit): Update in-app strings

### PR DESCRIPTION
**Pull request by KobeW50**
Complementary to https://github.com/inotia00/revanced-patches/pull/40

### CODE REVIEW

File: app/src/main/java/app/revanced/reddit/settings/SettingsEnum.java [hunk 1]
```diff
@@ -19,38 +19,38 @@ public enum SettingsEnum {
...
-            "Hide discover button or communities button in navigation."),
+            "Hides the discover or communities button in the navigation bar."),
     HIDE_RECENTLY_VISITED_SHELF("revanced_hide_recently_visited_shelf", BOOLEAN, FALSE,
             "Hide recently visited shelf",
-            "Hides recently visited shelf in sidebar."),
+            "Hides the recently visited shelf in the sidebar."),
     HIDE_TOOLBAR_BUTTON("revanced_hide_toolbar_button", BOOLEAN, FALSE, true,
             "Hide toolbar button",
-            "Hides 'r/place' button or 'reddit recap' button in the toolbar."),
+            "Hides the \"r/place\" or \"reddit recap\" button in the toolbar."),
```
Comment (KobeW50) [inline app/src/main/java/app/revanced/reddit/settings/SettingsEnum.java]: uh I hope I escaped the string correctly

File: app/src/main/java/app/revanced/reddit/settings/SettingsEnum.java [hunk 2]
```diff
@@ -19,38 +19,38 @@ public enum SettingsEnum {
...
     OPEN_LINKS_DIRECTLY("revanced_open_links_directly", BOOLEAN, TRUE,
             "Open links directly",
-            "Skips over redirection URLs to external links."),
+            "Skips over redirection URLs in external links."),
```
Comment (KobeW50) [inline app/src/main/java/app/revanced/reddit/settings/SettingsEnum.java:47]: I think this change is correct but not 100% sure

**Comment by inotia00** [comment]
I tested it a while ago, and it seems to have been changed as it is intended.

**Comment by KobeW50** [comment]
> I tested it a while ago, and it seems to have been changed as it is intended.

Wdym?

Also, I forgot to ask if "reddit recap button" on line 44 should be capitalized. Like:

`"Hides the \"r/place\" or \"Reddit recap\" button in the toolbar."`

**Comment by inotia00** [comment]
What I meant was that the escape character was reflected normally

The attached screenshot is the result of building this PR

RRR

All you have to do is choose whether to capitalize `reddit recap` or not

**Comment by KobeW50** [comment]
I realized that the Reddit recap button is just an icon button without a label, so I removed the double quotes surrounding it.

**Comment by inotia00** [comment]
To reflect the changes in the private repository (dev branch), I will merge this PR first

If you have any additional changes, please open a new PR

**Pull request merged**